west.yml

```diff
@@ -98,7 +98,7 @@ manifest:
       groups:
         - hal
     - name: hal_nxp
-      revision: 66db7c2a57ec1adfd3574169346ed3af9bf17c0b
+      revision: pull/131/head
```

Comment:
this commit should go first for bisectability

Reply:
Thanks, done.